.gitlab-ci.yml

```diff
@@ -21,6 +21,23 @@ generate-scripts:
       - install_script_agent6.sh
       - install_script_agent7.sh
 
+test-opensuse-13:
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/docker_x64:$DATADOG_AGENT_BUILDIMAGES
+  tags: ["runner:docker"]
+  stage: test
+  dependencies: ["generate-scripts"]
+  parallel:
+    matrix:
+      - IMAGE: images/mirror/opensuse/archive:13.2
```

Comment:
As I did this modification on my PR, I suggest not to repeat `images/mirror/` for all images in a matrix and set this directly on line 37: `--image "registry.ddbuild.io/images/mirror/${IMAGE}`